source/ini_funcs.hpp

```diff
@@ -132,6 +132,9 @@ PackageHeader getPackageHeaderFromIni(const std::string& filePath) {
                 // Value not enclosed in quotes
                 packageHeader.github = strLine.substr(repoPos, endPos - repoPos);
             }
+            if (packageHeader.github.substr(packageHeader.github.length() - 11, 11) != "?per_page=1") {
```

Comment:
Better to use `ends_with` because it does not generate an additonal string
```suggestion
            if (packageHeader.github.ends_with("?per_page=1")) {
```